packages/client/components/ScopePhaseAreaGitLab.tsx

```diff
@@ -0,0 +1,62 @@
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {ScopePhaseAreaGitLab_meeting$key} from '../__generated__/ScopePhaseAreaGitLab_meeting.graphql'
+import ScopePhaseAreaAddGitLab from './ScopePhaseAreaAddGitLab'
+
+interface Props {
+  isActive: boolean
+  gotoParabol: () => void
+  meetingRef: ScopePhaseAreaGitLab_meeting$key
+}
+
+graphql`
+  fragment ScopePhaseAreaGitLab_teamMember on TeamMember {
```

Comment:
-1 this fragment should be added to the mutation response here: https://github.com/ParabolInc/parabol/blob/918b002b3315f0c5cfba9e5b5e50c80ed17ccdf9/packages/client/mutations/AddTeamMemberIntegrationAuthMutation.ts#L9